### DISCUSSION TITLE
Can I remove an app from sale with fastlane on App Store Connect?

### DISCUSSION THREAD

**Discussion by lawliet616** [fastlane/fastlane#17032] [Q&A]
Hey,

On ASC when you edit an app there's an Availability section under General > Pricing and Availability. 
Is there a way to uncheck every country and region and the "New Countries or Regions" box?

I'm aware that you can update territory ids with fastlane like this:
```
attributes = {
    :availableInNewTerritories => false
}
app.update(attributes: attributes, territory_ids: [])
```
But if you pass an empty array it is just ignored: https://github.com/fastlane/fastlane/blob/35de7256c2c6df0e5cbf45510703d4a21829229c/spaceship/lib/spaceship/connect_api/tunes/tunes.rb#L125

**Comment by lucgrabowski** [comment]
I've opened a PR to allow for removing an app from sale: https://github.com/fastlane/fastlane/pull/20324
